data/tlg0627/tlg001/__cts__.xml

```diff
@@ -1,7 +1,7 @@
 <ti:work xmlns:ti="http://chs.harvard.edu/xmlns/cts" groupUrn="urn:cts:greekLit:tlg0627" xml:lang="grc" urn="urn:cts:greekLit:tlg0627.tlg001">
-  <ti:title xml:lang="eng">De prisca medicina</ti:title>
+  <ti:title xml:lang="lat">De prisca medicina</ti:title>
   <ti:edition urn="urn:cts:greekLit:tlg0627.tlg001.1st1K-grc1" workUrn="urn:cts:greekLit:tlg0627.tlg001">
-    <ti:label xml:lang="eng">De prisca medicina</ti:label>
-    <ti:description xml:lang="eng">Hippocrates, De prisca medicina</ti:description>
+    <ti:label xml:lang="lat">De prisca medicina</ti:label>
+    <ti:description xml:lang="mul">Hippocrates, De prisca medicina, Oeuvres complètes, Littré, Volume 1, 1839</ti:description>
```

Comment:
The work title does not need to be repeated in the description, just the book/edition name. So,  
`Hippocrates. Oeuvres complètes d'Hippocrate, Vol. 1.  Littré, Émile, editor. Paris: Baillière, 1839.`

Reply:
This practice has been inconsistent across the CTS files over times, I didn't actually add the work titles to the description, I just didn't delete them, but I can!

Reply:
Right, I figured if you were here, you'd want to adopt a more standard format.
You don't have to, of course!  I can fix this later on the pull request.